packages/manager/cypress/e2e/core/loadBalancers/load-balancer-configurations.spec.ts

```diff
@@ -43,75 +51,406 @@ describe('Akamai Global Load Balancer configurations page', () => {
+      }
     });
+  });
+  describe('delete', () => {
+    it('deletes a configuration', () => {
+      const loadbalancer = loadbalancerFactory.build();
+      const configurations = configurationFactory.buildList(1);
+
+      mockGetLoadBalancer(loadbalancer).as('getLoadBalancer');
+      mockGetLoadBalancerConfigurations(loadbalancer.id, configurations).as(
+        'getConfigurations'
+      );
+
+      cy.visitWithLogin(`/loadbalancers/${loadbalancer.id}/configurations`);
+
+      cy.wait([
+        '@getFeatureFlags',
+        '@getClientStream',
+        '@getLoadBalancer',
+        '@getConfigurations',
+      ]);
+
+      const configuration = configurations[0];
+
+      const accordionHeader = cy.findByText(configuration.label);
+
+      // Click the accordion header to open the accordion
+      accordionHeader.click();
+
+      // Get the Configuration's entire accordion area
+      accordionHeader.closest('[data-qa-panel]').within(() => {
+        // Click the Delete button to open the delete dialog
+        ui.button.findByTitle('Delete').click();
+      });
+
+      mockDeleteLoadBalancerConfiguration(loadbalancer.id, configuration.id).as(
+        'deleteConfiguration'
+      );
+      mockGetLoadBalancerConfigurations(loadbalancer.id, []).as(
+        'getConfigurations'
+      );
+
+      ui.dialog
+        .findByTitle(`Delete Configuration ${configuration.label}?`)
+        .within(() => {
+          cy.findByText(
+            'Are you sure you want to delete this configuration?'
+          ).should('be.visible');
+
+          ui.button.findByTitle('Delete').click();
+        });
 
-    ui.button
-      .findByTitle('Create Configuration')
-      .should('be.visible')
-      .should('be.enabled')
-      .click();
+      cy.wait(['@deleteConfiguration', '@getConfigurations']);
+
+      cy.findByText(configuration.label).should('not.exist');
+    });
+    it('shows API errors when deleting a configuration', () => {
+      const loadbalancer = loadbalancerFactory.build();
+      const configurations = configurationFactory.buildList(1);
+
+      mockGetLoadBalancer(loadbalancer).as('getLoadBalancer');
+      mockGetLoadBalancerConfigurations(loadbalancer.id, configurations).as(
+        'getConfigurations'
+      );
+
+      cy.visitWithLogin(`/loadbalancers/${loadbalancer.id}/configurations`);
+      cy.wait([
+        '@getFeatureFlags',
+        '@getClientStream',
+        '@getLoadBalancer',
+        '@getConfigurations',
+      ]);
+
+      const configuration = configurations[0];
+
+      const accordionHeader = cy.findByText(configuration.label);
+
+      // Click the accordion header to open the accordion
+      accordionHeader.click();
+
+      // Get the Configuration's entire accordion area
+      accordionHeader.closest('[data-qa-panel]').within(() => {
+        // Click the Delete button to open the delete dialog
+        ui.button.findByTitle('Delete').click();
+      });
```

Comment:
```suggestion
      cy.findByText(configuration.label).as('accordionHeader');
      // Click the accordion header to open the accordion
      cy.get('@accordionHeader').click();
      // Get the Configuration's entire accordion area
      cy.get('@accordionHeader')
        .closest('[data-qa-panel]')
        .within(() => {
          // Click the Delete button to open the delete dialog
          ui.button.findByTitle('Delete').click();
        });
```
We want to [steer clear of assigning selected elements to variables or constants](https://docs.cypress.io/guides/references/best-practices#Assigning-Return-Values) because if the element were to be re-rendered or otherwise change between the time that the const is assigned and when you try to interact with it, Cypress's reference to the element will be invalid and the test will fail.

If the goal is to avoid having to call `cy.findByText(configuration.label)` repeatedly, assigning it as an alias should help reduce the repetition without the risk of Cypress's reference to the element becoming invalid.